csharp/src/Google.Protobuf/Reflection/FieldDescriptor.cs

```diff
@@ -168,14 +168,16 @@ public bool IsMap
             get { return fieldType == FieldType.Message && messageType.Proto.Options != null && messageType.Proto.Options.MapEntry; }
         }
 
-        // TODO(jonskeet): Check whether this is correct with proto3, where we default to packed...
-
         /// <summary>
         /// Returns <c>true</c> if this field is a packed, repeated field; <c>false</c> otherwise.
         /// </summary>
         public bool IsPacked
```

Comment:
looks like this is only used in WireFormat.MakeTag(int, FieldDescriptor). The generated code already has tags inlined, so what is this going to be used for? DynamicMessage? In that case we should make sure we fail early once someone tries to push us a descriptor for proto2 message (but I think we are already doing that).

Just wondering if it wouldn't be fine to just hide this field.